include/libp2p/protocol/kademlia/error.hpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LIBP2P_PROTOCOL_KADEMLIA_ERROR
+#define LIBP2P_PROTOCOL_KADEMLIA_ERROR
+
+#include <libp2p/outcome/outcome.hpp>
+
+namespace libp2p::protocol::kademlia {
+  enum class Error {
+    SUCCESS = 0,
```

Comment:
Docs say we must not use zero value item as it is already implicitly used for the success case.
https://kagome.readthedocs.io/en/latest/development/result.html

Why not to use return ```outcome::success()``` in code instead of returning ```Error::SUCCESS```?

Reply:
Hm... This is realy not used. Will be removed.